addons/event/models/event.py

```diff
@@ -179,6 +179,12 @@ def _compute_auto_confirm(self):
     badge_innerleft = fields.Html(string='Badge Inner Left')
     badge_innerright = fields.Html(string='Badge Inner Right')
     event_logo = fields.Html(string='Event Logo')
+    schedule_change = fields.Boolean()
```

Comment:
ok. purpose of adding this field to hide/show alert box in event form view + send notification to attendee regarding schedule updated.

can you change field name: event_schedule_updated ?